packages/ui/src/components/ui-prompt.tsx

```diff
@@ -15,44 +15,48 @@ import type { FormEvent, ReactNode } from 'react'
 import { useId, useState } from 'react'
 
 interface UiPromptProps {
-  title: ReactNode
-  description: ReactNode
-  label: ReactNode
-  placeholder?: string
-  actionLabel: ReactNode
   action: (value: string) => void
+  actionLabel: ReactNode
   children: ReactNode
+  description?: ReactNode | undefined
+  label: ReactNode
+  placeholder?: string
+  title: ReactNode
+  value?: string | undefined
 }
 
-export function UiPrompt({ title, description, label, placeholder, actionLabel, action, children }: UiPromptProps) {
+export function UiPrompt({
+  action,
+  actionLabel,
+  children,
+  description,
+  label,
+  placeholder,
+  title,
+  value: inputValue = '',
+}: UiPromptProps) {
   const [isOpen, setIsOpen] = useState(false)
-  const [value, setValue] = useState('')
+  const [value, setValue] = useState(inputValue)
```

Comment:
The `UiPrompt` component needed changed to accommodate for prompts with default text.